ros2bag/ros2bag/verb/list.py

```diff
@@ -27,8 +27,9 @@ class ListVerb(VerbExtension):
 
     def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
-            'plugin_type', help='lists available plugins (storage plugins or converter plugins',
-            choices=['storage', 'converter'])
+            'plugin_type',
+            help='lists available plugins',
+            choices=['storage', 'converter', 'compression', 'decompression'])
```

Comment:
I think I'd like to call these `compressor` and `decompressor` to match the internal interface names and `converter` (which isn't `conversion`)